src/transaction/log_tran_table.c

```diff
@@ -1526,7 +1526,6 @@ logtb_clear_tdes (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
   LSA_SET_NULL (&tdes->topop_lsa);
   LSA_SET_NULL (&tdes->tail_topresult_lsa);
   LSA_SET_NULL (&tdes->commit_abort_lsa);
-  tdes->last_mvcc_lsa.set_null ();
```

Comment:
Is mvccinfo.reset() not needed here?

Reply:
Yes, it would be needed.
But it would also mean a - rather drastic - change in functionality.
Below, in the same function, there is an
```
  assert (tdes->mvccinfo.id == MVCCID_NULL);
```
This assert was introduced with patch 3d49b48ce9 in 2015 for http://jira.cubrid.org/browse/CUBRIDSUS-15914 .

So it seems that the invariant is that the MVCC information must be already cleaned when entering `logtb_clean_tdes`.
I would maintain the existing functionality as we don't have an overview of what would mean to change this invariant.

So best it would be only clean the `mvcc_info.last_mvcc_lsa`.
It's ugly but risky otherwise.

Reply:
Other option is to remove this `set_null` as already done.
And add an assert of ours together with the existing one:
```
assert (tdes->mvccinfo.id == MVCCID_NULL);
assert (tdes->mvccinfo.last_mvcc_lsa.is_null ());
```
And we'll see how this fares in tests.

Reply:
Thanks for detail comment. 

https://github.com/CUBRID/cubrid/pull/3839/files#r989675406 
this comment could be an answer for the feedback. 

In the process of starting a transaction, performing mvcc operation, finishing mvcc, and clearing tdes, 
the above assert statement will work well. 

However, during recovery, only mvccid is initialized among mvcc information and then tdes is initialized. 
Then, the uninitialized mvcc information (last_mvcc_lsa) is used in the transaction that reused the corresponding tdes, causing a problem. 

So, I replace the place where only mvccid is initialized with mvccinfo.reset (), 
and it seems okay to put this into logtb_clear_tdes ().

Reply:
To reach a middle ground in `logtb_clear_tdes`:
- to keep the existing invariant of the assert:
```
assert (tdes->mvccinfo.id == MVCCID_NULL);
```
- to make sure that mvccinfo is still initialized
I would change to the following:
```
assert (tdes->mvccinfo.id == MVCCID_NULL);
tdes->mvccinfo.reset ();
```

The root problem is perhaps that the route `logtb_rv_find_allocate_tran_index` -> `logtb_allocate_tran_index` calls `logtb_clear_tdes`. It should, instead, call `logtb_initialize_tdes` which seems more appropriate:
- `logtb_initialize_tdes` is to initialize an empty/garbage structure
- `logtb_clear_tdes` is to clear an initialized structure